MoltenVK/MoltenVK/Commands/MVKCmdTransfer.mm

```diff
@@ -1138,6 +1139,7 @@ static inline MTLSize mvkClampMTLSize(MTLSize size, MTLOrigin origin, MTLSize ma
 		VkRect2D renderArea = cmdEncoder->clipToRenderArea({{0, 0}, fbExtent});
 		fbExtent = {renderArea.offset.x + renderArea.extent.width, renderArea.offset.y + renderArea.extent.height};
 	}
+#endif
 	[tempRPDesc release];													// temp release
```

Comment:
Need to include `[tempRPDesc release]` before `#endif`.

Reply:
Dammit, I thought I did that. #1065